osu.Game/Screens/Edit/Editor.cs

```diff
@@ -109,6 +109,12 @@ private void load(OsuColour colours, GameHost host, OsuConfigManager config)
             beatDivisor.Value = Beatmap.Value.BeatmapInfo.BeatDivisor;
             beatDivisor.BindValueChanged(divisor => Beatmap.Value.BeatmapInfo.BeatDivisor = divisor.NewValue);
```

Comment:
this also accesses `Beatmap.Value`, which I assume was the whole root cause of the issue (clock was being created using dummy beatmap instead of new one) - maybe the beatmap creation code should be at the very start of `load()` for complete sanity?

Reply:
hmm, good catch. this just means that the divisor won't be saved/restored between sessions, but still worth fixing.